src/Command/OrderConfirmedCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Doctrine\ORM\EntityManagerInterface;
+use App\Repository\OrderRepository;
+use App\Service\MailService;
+use Symfony\Component\Routing\RouterInterface;
+
+#[AsCommand(
+    name: 'app:order-confirmed',
+    description: 'Add a short description for your command',
```

Comment:
please be more descriptive and mention what this command will do.

Reply:
done